.gitattributes

```diff
@@ -0,0 +1,18 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.md text
+*.txt text
+*.yaml text
+
+# Declare files that will always have LF line endings on checkout.
+*.sh text eol=lf
+*.conf text eol=lf
+*.properties text eol=lf
```

Comment:
Will we also need to include .gradle files here?

Reply:
Thankfully no Gradle in this repo!